### PR TITLE
docs: add info about the edge compatibility

### PR DESCRIPTION
**Pull request by makma**
Add info about SDK's compatibility.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -59,6 +59,16 @@ client.getVisitorHistory("<visitorId>").then(visitorHistory => {
 const visit = visitWebhookBody as unknown as VisitWebhook;
 ```
 
+### Compatibility
+This SDK might be also compatible, however not actively tested, with some runtimes that are incompatible with the NodeJS environment (such as [NextJS Edge](https://beta.nextjs.org/docs/rendering/edge-and-nodejs-runtimes#edge-runtime) runtime). The proper behavior might be achieved by passing the environment's native `fetch()` function with the proper binding:
```
Comment (JuroUhlar) [inline readme.md:63]:
```suggestion
This SDK might be also compatible, however not actively tested, with JavaScript "edge" runtimes that do not support all Node APIs, for example, [Vercel Edge Runtime](https://edge-runtime.vercel.app/), or [Cloudflare Workers](https://developers.cloudflare.com/workers/). 

To make it work, replace the SDK's built-in fetch function (which relies on Node APIs) with the runtime's native `fetch` function. Pass the function into the constructor with proper binding: 
```

- Tweak for more specificity
- Link to Vercel, not Next docs (the suggestion is presumably also applicable to Remix, SvelteKit, etc apps running on Vercel)
- Add Cloudflare workers as an example

**Comment by makma** [comment]
Thank you, @JuroUhlar, for the improvements!

**Review by ilfa — APPROVED** [review]
